tests/generic_config_updater/test_empty_input.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import pytest
+
+from tests.generic_config_updater.gu_utils import apply_patch, expect_op_success
+from tests.generic_config_updater.gu_utils import generate_tmpfile, delete_tmpfile
+from tests.generic_config_updater.gu_utils import create_checkpoint, delete_checkpoint, rollback_or_reload
+
+logger = logging.getLogger(__name__)
+
+@pytest.fixture(autouse=True)
+def setup_env(duthosts, rand_one_dut_hostname):
```

Comment:
>setup_env [](http://example.com/codeflow?start=4&length=9)

Seems that you are assume there is some pre-condition before you run this test case.

Suggest make the pre-condition of invalid configDB explicit. So it will always be tested in a reproducible fashion.